docs/modules/configuration/partials/fullconfiguration.adoc

```diff
@@ -74,6 +74,22 @@ engine:
   defaultPolicyVersion: "default" # DefaultPolicyVersion defines what version to assume if the request does not specify one.
   globals: {"environment": "staging"} # Globals are environment-specific variables to be made available to policy conditions.
   lenientScopeSearch: false # LenientScopeSearch configures the engine to ignore missing scopes and search upwards through the scope tree until it finds a usable policy.
+hub:
+  connection: # Connection holds advanced connection settings for Cerbos Hub.
```

Comment:
I feel like "advanced" implies optional and that the defaults should suffice for most users, but then the endpoints are required. Should we change that so that they have the suggested values as defaults?

Reply:
Actually, this section is not meant to be configured by users at all. Should we just hide it?

Reply:
Yeah, I think it's reasonable for this section to be an undocumented escape hatch.

Reply:
I think the `connection` section still needs an ignore tag on it.

Reply:
Lost in the git shuffle 🤦🏽